chapters/inheritance.tex

```diff
@@ -983,60 +983,107 @@ \subsection{Restrictions on Redeclarations}\label{restrictions-on-redeclarations
   \end{nonnormative}
 \end{itemize}
 
-\subsection{Annotation Choices for Suggested Redeclarations and Modifications}\label{annotation-choices-for-suggested-redeclarations-and-modifications}
+\subsection{Annotations for Suggested Redeclarations and Modifications}\label{annotations-for-suggested-redeclarations-and-modifications}
```

Comment:
I checked it, and only the last `\label` is turned into the `id` of a `section` tag in the HTML.  Hence, we should only have one `\label`, and it would be good if it was more stable than the exact formulation of the section title.